frontend/src/Frontend/UI/Dialogs/AddAccount.hs

```diff
@@ -78,19 +83,31 @@ uiCreateWalletStepOne
   -> Event t ()
   -> Workflow t m (Text, (mConf, Event t ()))
 uiCreateWalletStepOne model onClose = Workflow $ do
+  pb <- getPostBuild
+  let dInflightAcc = model ^. wallet_accounts . to (fmap findFirstInflightAccount)
+
   (dSelectedChain, dNotes, onAddVanityAcc) <- modalMain $ do
-    divClass "segment modal__main transaction_details" $ do
+      dyn_ $ ffor dInflightAcc $ \case
+        Nothing -> blank
+        Just _ -> divClass "segment modal__main transaction_details" $ do
+          elClass "h2" "heading heading_type_h2" $ text "Notice"
+          divClass "group segment" $ el "p"
+            $ text "An incomplete vanity account was detected and we were unable to verify its existence on the chain. The known details have been pre-populated for you to complete vanity account creation"
+
       elClass "h2" "heading heading_type_h2" $ text "Destination"
       dChainId <- divClass "group segment" $ do
         transactionDisplayNetwork model
-        userChainIdSelect model
+        userChainIdSelectWithPreselect model (fmap _account_chainId <$> dInflightAcc)
 
       elClass "h2" "heading heading_type_h2" $ text "Reference Data"
-      dNotes <- divClass "group segment" $
-        value <$> mkLabeledClsInput True "Notes" inpElem
+      dNotes <- divClass "group segment" $ fmap value $ mkLabeledClsInput True "Notes" $ inpElem $ def
+        & inputElementConfig_setValue .~ tagMaybe (fmap (unAccountNotes . _account_notes) <$> current dInflightAcc) pb
```

Comment:
If an inflight account is found *after* this dialog was opened, it seems to me that the above `dyn_` would make us claim the details were pre-populated, but at least this field won't react?